docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -504,10 +515,6 @@ children:
     url: express-with-lb4-rest-tutorial.html
     output: 'web, pdf'
 
-  - title: 'How to secure your LoopBack 4 application with JWT authentication'
```

Comment:
Is it ok to have duplicate entries in the sidebar? This is one of the tutorials, so I guess it might be better to leave it there too?

Reply:
sure